app/views/layouts/_flash_notices.html.erb

```diff
@@ -4,7 +4,7 @@
 
   <% if name == 'notice' %>
     <div class='flash-holder'>
-      <%= content_tag :p, msg, class: "govuk-heading-m flash-message-notice" %>
+      <%= tag.p(msg, class: "govuk-heading-m flash-message-notice") %>
```

Comment:
@alan-gds Can you provide more context please as to why `tag.p` is preferred here?

Reply:
Apparently content_tag is legacy syntax.
https://github.com/rubocop-hq/rubocop-rails/pull/242